Google.Cloud.Speech.V1.Samples/SpeechTranscribeAsync.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("LongRunningRequestPollUntilComplete", "speech_transcribe_async")
+
+// sample-metadata
+//   title: Transcribe Audio File using Long Running Operation (Local File) (LRO)
+//   description: Transcribe a long audio file using asynchronous speech recognition
+//   usage: dotnet run [--local_file_path "resources/brooklyn_bridge.raw"]
+
+using CommandLine;
+
+namespace Google.Cloud.Speech.V1.Samples
+{
+    // [START speech_transcribe_async]
+    using Google.Cloud.Speech.V1;
+    using Google.Protobuf;
+    using System;
+    using System.IO;
+
+    public class SpeechTranscribeAsync
+    {
+        /// <summary>
+        /// Transcribe a long audio file using asynchronous speech recognition
+        /// </summary>
+        /// <param name="localFilePath">Path to local audio file, e.g. /path/audio.wav</param>
+        public static void SampleLongRunningRecognize(string localFilePath)
+        {
+            SpeechClient speechClient = SpeechClient.Create();
+            // string localFilePath = "resources/brooklyn_bridge.raw"

```

Comment:
I'm not keen on this commented-out line. I think the parameter is clear enough. (And the same in general for all samples.)

Reply:
Actually if someone copy-pastes this code and uncomments that line, the code won't even compile.
You can use default parameter values or XML documentation on the method if you want to write an example of how it should look.
```
public static void SampleLongRunningRecognize(
    string localFilePath="resources/brooklyn_bridge.raw")
{
    ...
}
```

Reply:
+💯

/cc @SurferJeffAtGoogle